api/v1/values_types_extra.go

```diff
@@ -30,6 +31,114 @@ type CNIValues struct {
 	Global *CNIGlobalConfig `json:"global,omitempty"`
 }
 
+// Configuration for CNI.
```

Comment:
Okay, so you are moving this from auto-generated struct (using upstream Operator APIs) to this file. Did we try adding the env' params to the upstream CNIConfig struct and met with some resistance? Looping @dgn for his thoughts on this.

Reply:
@sridhargaddam 
Somehow, I didn't think about trying to add it upstream first.
Let me look into this.
That's definitely a preferred way.

Reply:
@sridhargaddam 

The PR has been approved and merged.
I'll make the verification to see how to modify this PR according to the merged changes..

Reply:
Thats great to hear @MaxBab.